File: Submissions/T19/M2/m2.py
n = int(input("Masukkan N : "))
a, b = 0, 1
seq = []
for _ in range(n):
    seq.append(a)
    a, b = b, a + b
print("Deret :", *seq)
print(f"Fibonacci ke-{n} : {seq[-1]}")

File: Submissions/T19/M2/m2.c
#include <stdio.h>
int main() {
    int n;
    printf("Masukkan N : "); scanf("%d", &n);
    int a = 0, b = 1, tmp, last = 0;
    printf("Deret :");
    for (int i = 0; i < n; i++) {
        printf(" %d", a);
        last = a;
        tmp = a + b; a = b; b = tmp;
    }
    printf("\nFibonacci ke-%d : %d\n", n, last);
}
